core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -130,4 +130,21 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
   def getCreateAzureRelayResult(workspaceId: UUID, jobControlId: String, accessToken: OAuth2BearerToken): CreateControlledAzureRelayNamespaceResult = {
     getControlledAzureResourceApi(accessToken).getCreateAzureRelayNamespaceResult(workspaceId, jobControlId)
   }
+
+  def createAzureStorageAccount(workspaceId: UUID, region: String, accessToken: OAuth2BearerToken) = {
+    // Storage account names must be unique and 3-24 characters in length, numbers and lowercase letters only.
+    val prefix = workspaceId.toString.substring(0, workspaceId.toString.indexOf("-"))
+    val suffix = workspaceId.toString.substring(workspaceId.toString.lastIndexOf("-") + 1)
+    getControlledAzureResourceApi(accessToken).createAzureStorage(
+      new CreateControlledAzureStorageRequestBody().common(
+        new ControlledResourceCommonFields().name(s"storage-rcf-${workspaceId}").
+          cloningInstructions(CloningInstructionsEnum.NOTHING).
```

Comment:
I'm not sure if these are the correct values (thought they are what I used for the relay).

Reply:
This should be ok.